package/MDAnalysis/coordinates/PDB.py

```diff
@@ -384,6 +384,8 @@ def _read_frame(self, frame):
             self.convert_pos_from_native(self.ts._unitcell[:3])
         self.ts.frame = frame
         self.ts.data['occupancy'] = occupancy
+        for aux in self.aux_list:
```

Comment:
@kain88-de I'm not sure what you mean, sorry?
@richardjgowers I've just made the issue now :)